packages/shared/forks/ReactFeatureFlags.testing.js

```diff
@@ -23,7 +23,7 @@ export const enableUpdaterTracking = false;
 export const enableSuspenseServerRenderer = false;
 export const enableSelectiveHydration = false;
 export const enableLazyElements = false;
-export const enableCache = false;
+export const enableCache = true;
```

Comment:
Probably should have turned this on earlier but didn't matter before.

Reply:
Should this be set to `__EXPERIMENTAL__`?